package.json

```diff
@@ -67,7 +67,7 @@
   },
   "scripts": {
     "dev": "vite",
-    "start": "yarn dev",
+    "start": "yarn dev --host 127.0.0.1",
```

Comment:
Maybe we can put that setting in the vite.config.ts file and it could be changed by an env variable ? To make it more flexible.